src/js/components/Peak/PeakPosts.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Post as PostType } from '../../models/interfaces';
+import { getPeakPostsPaginated } from '../../API/peaks/methods';
+import { Post } from '../Home';
+import styles from './PeakPosts.module.css';
+import postStyles from '../Home/Posts.module.css';
```

Comment:
Nie podoba mi się importowanie css-modules z innego komponentu, rozmija się to kompletnie z ideą izolacji cssów.  Można albo zduplikować jakiś kod css, albo spróbować wydzielić wspólny komponent który zostanie wykorzystany zarówno tutaj jak i w Posts

Reply:
Wydaje mi się że wystarczy przerobić Posts na common component bo z tego co widzę to przyda się przynajmniej w 3 miejscach, a będzie to bardzo proste.

Reply:
Nie chciałam po prostu powtarzać kodu, ale zamienię Posts na common component.